src/applications/vaos/tests/new-appointment/components/TypeOfAudiologyCare.unit.spec.jsx

```diff
@@ -61,7 +62,10 @@ describe('VAOS <TypeOfAudiologyCarePage>', () => {
       { store },
     );
 
-    fireEvent.click(await screen.findByLabelText(/routine hearing/i));
+    userEvent.click(await screen.findByLabelText(/routine hearing/i));
+    await waitFor(() => {
+      expect(screen.getByLabelText(/routine hearing/i).checked).to.be.true;
```

Comment:
This hasn't worked reliably in the past, I think because of the issue identified here, where we're checking a form input before Redux can properly handle it.